packages/generator-single-spa/src/react/templates/package.json

```diff
@@ -11,7 +11,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "pretty-quick --staged && concurrently <%= packageManager %>:test <%= packageManager %>:lint <%= packageManager %>:typescript <%= packageManager %>:coverage"
```

Comment:
Copy/paste error with typescript stuff. And no need to run both test and coverage during pre-commit